packages/babel-parser/src/plugins/typescript/index.js

```diff
@@ -2365,7 +2365,13 @@ export default (superClass: Class<Parser>): Class<Parser> =>
       }
     }
 
-    toAssignableList(exprList: N.Expression[]): $ReadOnlyArray<N.Pattern> {
+    toAssignableList(
+      exprList: N.Expression[],
+      isBinding: ?boolean,
+      contextDescription: string,
+    ): $ReadOnlyArray<N.Pattern> {
+      const isAssignmentExpression =
+        contextDescription === "assignment expression";
```

Comment:
I would prefer not to rely on the exact value of this string, since it's only used for error messages. Could we use `!isBinding` instead of `isAssignmentExpression`?

Reply:
Highly agree on not using the value of the string. What does `isBinding` represent? Can you point me to any doc or link?